desc/objectives/_omnigenity.py

```diff
@@ -163,6 +169,7 @@ def build(self, use_jit=True, verbose=1):
             transforms["B"].basis.modes,
             helicity=self.helicity,
             NFP=transforms["B"].basis.NFP,
+            NFP_umbilic_factor=transforms["B"].basis.NFP_umbilic_factor,
```

Comment:
As a test to see if we can get ANY QS umbilic (meaning just having a umbilic factor greater than one, not caring about sharp edges as even with smooth edges it is still a shape that repeates only every 2pi * umbliic factor times), can you try optimizing without the extra umbilic objective you have?

this would give confidence in the code changes you've made and that everything works fine with the umbilic factor, before adding other objectives we aren't sure of